client/src/components/profile/components/TimeLine.tsx

```diff
@@ -323,7 +323,7 @@ class TimelineInner extends Component<TimelineInnerProps, TimeLineInnerState> {
     );
   }
 }
-/* eslint-disable */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/restrict-template-expressions, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call*/
```

Comment:
Oh, I see.  Not a small number of them...

Reply:
You asked, and you received 😄